### ISSUE TITLE
How to clear sort during data refresh

### ISSUE THREAD

**Issue by smartgoo** [vincjo/datatables#150]
I'm hoping to clear both search and sort after loading new data (using v2 and Svelte 5). The following properly clears search. However, if a column is sorted prior to calling `setRows()`, that sort persists after `setRows()`:
```javascript
$effect(() => { 
    data;
    table.clearSearch();
    table.setRows(data) 
})
```
`clearSort()` is not listed as an available method on the docs site though it looks like #40 introduced it. Assuming that was in an older version though (am a new user to this library). Trying to use it yields `table.clearSort is not a function`.

Is there a different way to clear sort in v2? Maybe I'm off here

**Comment by vincjo** [comment]
Indeed, there is no way in V2. Thx for pointing it out.

`clearSort()` can be added easily.

**Comment by vincjo** [comment]
Just published on npm, `2.2.0`

Doc: https://vincjo.fr/datatables/api/client/methods~clearSort
